easy_db/database.py

```diff
@@ -153,7 +153,7 @@ def pull_table(self, tablename: str, columns='all', clear_cache=False, progress_
 
                 if progress_handler is not None:
                     if self.db_type == 'SQLITE3':  # progress_handler only currently working for sqlite
-                        conn.set_progress_handler(progress_handler, 100)  # Can use to track progress
+                        conn.set_progress_handler(*progress_handler if type(progress_handler) is tuple else (progress_handler, 100)) # Can use to track progress
```

Comment:
This is my solution to allowing the option to set the second positional argument.
If you want to set the N argument, you can pass progress_handler as a tuple, or else it'll set the second argument.

Reply:
This works BTW, I tested it

>>> t=(1,2)
>>> f(*t if type(t) is tuple else (2,3))
1, 2
>>> t=1
>>> f(*t if type(t) is tuple else (2,3))
2, 3

Reply:
This is a clever way to provide the n argument!  Only power users would be specifying this, so I think it's alright that the progress_handler arg could be in the form of single callback or a 2-tuple of form (callback, n).  Thanks for the code snippet test there; I was wondering on if a * was needed after the else for unpacking.

I also thought about using `if isinstance(progress_handler, tuple)`, but your `if type(progress_handler) is tuple` seems more natural and intuitive.